src/extension.ts

```diff
@@ -2,15 +2,19 @@ import * as store from './store'
 import { window, ExtensionContext, commands } from 'vscode'
 import { exists, findManifests, ensureDocumentsPortal } from './utils'
 import { promises as fs } from 'fs'
-import { FlatpakTaskTerminal, TaskMode } from './terminal'
+import { StatusBarItem } from './statusBarItem'
+import { FlatpakTaskTerminal } from './terminal'
```

Comment:
As you are touching this, maybe rename it to FlatpakTerminal ? :)

Reply:
its probably a good idea too to split up FlatpakManifest and/or Command to different file?

Reply:
It is, no idea why I have put everything in a single file in the past